frontend/src/metabase/collections/components/PinnedQuestionCard/PinnedQuestionCard.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { t } from "ttag";
+import { isPreviewEnabled, Item } from "metabase/collections/utils";
+import Visualization from "metabase/visualizations/components/Visualization";
+import Metadata from "metabase-lib/lib/metadata/Metadata";
+import { Bookmark, Collection } from "metabase-types/api";
+import PinnedQuestionLoader from "./PinnedQuestionLoader";
+import {
+  CardActionMenu,
+  CardRoot,
+  CardPreviewSkeleton,
+  CardStaticSkeleton,
+} from "./PinnedQuestionCard.styled";
+
+export interface PinnedQuestionCardProps {
+  item: Item;
+  collection: Collection;
+  metadata: Metadata;
+  bookmarks?: Bookmark[];
+  onCopy: (items: Item[]) => void;
+  onMove: (items: Item[]) => void;
+  onCreateBookmark?: (id: string, model: string) => void;
+  onDeleteBookmark?: (id: string, model: string) => void;
+}
+
+const PinnedQuestionCard = ({
```

Comment:
I renamed this from `PinnedChartCard` to `PinnedQuestionCard`. Git tells it's a new file but it isn't